services/core-web/src/components/Forms/AddTailingsForm.js

```diff
@@ -112,11 +113,11 @@ export const AddTailingsForm = (props) => (
       <Col md={12} xs={24}>
         <Form.Item>
           <Field
-            id="has_itrb"
-            name="has_itrb"
-            label="Independent Tailings Review Board*"
+            id="itrb_exemption_status_code"
+            name="itrb_exemption_status_code"
+            label="Independent Tailings Review Board Exemption*"
             component={RenderSelect}
```

Comment:
Update label here to: "Has Independent Tailings Review Board?"

Reply:
fixed.